src/components/TrendingFeedItem.js

```diff
@@ -0,0 +1,68 @@
+import { Link } from 'react-router-dom';
+import { numberWithCommas } from '../utils/helper';
+import PropTypes from 'prop-types';
+
+const TrendingFeedItem = ({ trendingFeedServerData }) => {
+  return (
+    <div className="bg-dark">
+      {trendingFeedServerData.map((trendingVideosData) => (
+        <div
+          key={trendingVideosData.id}
+          className="card bg-dark text-light flex-lg-row flex-sm-column align-items-sm-center">
+          <div style={{ textAlign: 'center' }}>
+            <video
+              className="m-lg-5"
+              controls
+              width={trendingVideosData.videoMeta.width / 2}
+              height={trendingVideosData.videoMeta.height / 2}
+              src={trendingVideosData.videoUrl}
+              frameBorder="0"
+              allowFullScreen
+              title="Embedded video"
+            />
+          </div>
+          <div
+            className="flex-column d-flex justify-content-center align-items-center w-100"
+            >
+            <h2 className="card-title text-center mb-2 ">{trendingVideosData.text}</h2>
+            <Link to={'/' + trendingVideosData.authorMeta.name}>
```

Comment:
Не критично, проте можна зробити ось так

```suggestion
            <Link to={`/${trendingVideosData.authorMeta.name}`}>
```